client/src/styles/global.scss

```diff
@@ -1,2 +1,31 @@
-@import "~@trussworks/react-uswds/lib/uswds.css";
 @import "~@trussworks/react-uswds/lib/index.css";
+@import "~@trussworks/react-uswds/lib/uswds.css";
+
+.j40FloatLogoLeft {
+  width: 100%;
+  padding-left: 50%;
+}
+
+.j40Footer {
+  font-family: "sans-serif";
+  font-size: 12px;
+  line-height: 24px;
+}
+
+.j40FooterAddressTitle {
+  line-height: 2.5;
+  font-weight: bold;
+  white-space: nowrap;
```

Comment:
Can you explain why this change was necessary? It's possible this is workaround-able with classes also

Reply:
removed